src/main/java/org/mule/service/scheduler/internal/config/ContainerThreadPoolsConfig.java

```diff
@@ -211,7 +223,7 @@ private static void validateNumber(String propName, long result, boolean allowZe
   private int cpuLightPoolSize = 2 * cores;
   private int ioQueueSize = 0;
   private int ioCorePoolSize = cores;
-  private int ioMaxPoolSize = (int) (cores + ((mem - 245760) / 5120));
+  private int ioMaxPoolSize = (int) max(2, cores + ((mem - 245760) / 5120));
```

Comment:
isn't this laying to the user? if the user specifies `1`.. won't this override that and change it to `2`? That's not necessarily a bad thing, but a log message should be printed if that's the case

Reply:
nop, this is the default for when the property does not exist in the config file